### DISCUSSION TITLE
Pulling Doppler Radar Images

### DISCUSSION THREAD

**Discussion by aalthouse** [weather-gov/api#267] [Q&A]
Hello,
I am wondering if the NWS is looking to being able to pull doppler radar images using the API. I currently use the radar.weather.gov to pull the radar image but since its being discontinued, will images be added to the API or is there already a way to pull images using the API.

Thank You!

**Comment by scadergit** [comment]
Geospacial services will be handled by the new Geoserver (behind the scenes that is developed by a different team). Similar to the current radar, the Geoserver is creating pre-rendered images, but serving them under the new radar application, currently at: https://preview-radar.weather.gov/ridge/lite.

**Comment by jfbauer432** [comment]
That new radar site is live.  It does have usable images, but the basemap isn't the best

I found that https://www.weather.gov/radarfaq#download_files mentions that there are geotiffs at  https://mrms.ncep.noaa.gov/data/RIDGEII.  These combined with another basemap could be rather good.  Although I have no idea at present on how to do that.
